### PR TITLE
Added inerface IEntity

### PR DESCRIPTION
**Pull request by famafka**
Added entities Character and Location

### CODE REVIEW

File: AniRPG.Domain/Entities/Location.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using AniRPG.Domain.Common;
+
+namespace AniRPG.Domain.Entities
+{
+    public class Location : IEntity
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public ICollection<Location> Transitions { get; private set;}
```
Comment (aangairbender) [inline AniRPG.Domain/Entities/Location.cs]: This collection must be initialized.
You should create constructor with code like:
`Transitions = new HashSet<Location>();`

**Review by aangairbender — CHANGES_REQUESTED** [review]

File: AniRPG.Domain/Entities/Location.cs (hunk 1 above)
Comment (famafka) [reply AniRPG.Domain/Entities/Location.cs]: thanks, fixed.

File: AniRPG.Domain/Entities/Location.cs [hunk 2]
```diff
@@ -10,6 +10,11 @@ public class Location : IEntity
         public int Id { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
-        public ICollection<Location> Transitions { get; private set;}
+        public ICollection<Location> Transitions { get; private set; }
+
+        Location ()
```
Comment (aangairbender) [inline AniRPG.Domain/Entities/Location.cs]: This should be public constructor

**Review by aangairbender — CHANGES_REQUESTED** [review]

File: AniRPG.Domain/Entities/Location.cs (hunk 2 above)
Comment (famafka) [reply AniRPG.Domain/Entities/Location.cs]: yep. now should be OK.

Comment (aangairbender) [reply AniRPG.Domain/Entities/Location.cs]: Nice

**Pull request merged**